### ISSUE TITLE
WARNING: Receive target exists at unexpected location: 

### ISSUE THREAD

**Issue by mattheys** [digint/btrbk#393]
Hi, I'm getting the following warning, it might be nothing but I can't find any other info on it.
`WARNING: Receive target of "/mnt/btrfs/.snapshots/audio/audio.20210506" exists at unexpected location: /mnt/backups/.snapshots/audio/firstAudioBackup`

/mnt/btrfs is my main btrfs data volume and /mnt/backups is my btrfs backup volume.

I performed the following to take a backup of the first snapshot of a subvolume so it wouldn't get rotated out by btrbk and I would always have the initial state of the volume.
`sudo btrfs subvolume snapshot -r /mnt/backups/.snapshots/audio/audio.20210506 /mnt/backups/.snapshots/audio/firstAudioBackup`

I see they share the same gen, is that why, did I somehow update the gen when I cloned a copy of the subvolume?
```
**ID 3879 gen 2353 top level 5 path .snapshots/audio/audio.20210506**
ID 5070 gen 1938 top level 5 path .snapshots/audio/audio.20210509
ID 6497 gen 1977 top level 5 path .snapshots/audio/audio.20210512
ID 6517 gen 2038 top level 5 path .snapshots/audio/audio.20210513
ID 6565 gen 2071 top level 5 path .snapshots/audio/audio.20210514
ID 6592 gen 2092 top level 5 path .snapshots/audio/audio.20210515
ID 6598 gen 2117 top level 5 path .snapshots/audio/audio.20210516
ID 6610 gen 2143 top level 5 path .snapshots/audio/audio.20210517
ID 6620 gen 2167 top level 5 path .snapshots/audio/audio.20210518
ID 6630 gen 2219 top level 5 path .snapshots/audio/audio.20210519
ID 6745 gen 2268 top level 5 path .snapshots/audio/audio.20210520
ID 6839 gen 2304 top level 5 path .snapshots/audio/audio.20210521
ID 6892 gen 2331 top level 5 path .snapshots/audio/audio.20210522
ID 6909 gen 2358 top level 5 path .snapshots/audio/audio.20210523
**ID 6914 gen 2353 top level 5 path .snapshots/audio/firstAudioBackup**
ID 6920 gen 2389 top level 5 path .snapshots/audio/audio.20210524
ID 6949 gen 2439 top level 5 path .snapshots/audio/audio.20210525
ID 7034 gen 2440 top level 5 path .snapshots/audio/audio.20210526
```

Other info 

`btrbk command line client, version 0.31.2`
`btrfs-progs v5.4.1`

**Comment by digint** [comment]
It's nothing of concern: btrbk found out that you already have a subvolume on your target at some place unknown to btrbk. The intention of this warning is to give you a hint that you might get duplicates.
So it's hinting you the ovious, as you explicitely wanted this (keeping a duplicate, untouched by btrbk).

Note to self: this has caused too much trouble already, confusing people more than being useful. Will probably downgrade this to INFO, and make an option to re-enable it.

(and sorry for the late reply, I'm very busy these days...)

**Comment by mattheys** [comment]
No problem about the timing of the reply, it wasn't a high priority issue, do you want me to keep this open if you plan to do something about it or shall I close it?

**Comment by digint** [comment]
fixed in: 6718dc2f483229c2eb4569e1e788cb436048a0c2

will close the issue when new version is released, thanks!

**Comment by digint** [comment]
included in btrbk-0.31.3
